src/Shared/AzureManaged/GrpcRetryPolicyDefaults.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Grpc.Core;
+using Grpc.Net.Client.Configuration;
+
+namespace Microsoft.DurableTask;
+
+/// <summary>
+/// Provides default retry policy configurations for gRPC client connections.
+/// </summary>
+sealed class GrpcRetryPolicyDefaults
+{
+    /// <summary>
+    /// Default maximum number of retry attempts.
```

Comment:
I think updating the summary/description to contain the default value is better for clarity.  Same for below
```suggestion
    /// The maximum number of retry attempts by default. This value is set to 10.

```